windows/build-ruby

```diff
@@ -149,6 +149,7 @@ header "Extracting Ruby..."
 	echo "+ In $OUTPUT_DIR:"
 	cd "$OUTPUT_DIR"
 	echo "+ 7z x $CACHE_DIR/$RUBY_FILE"
+	# brew install 7zip (on mac) and run as 7zz
```

Comment:
This should be pulled out into some developer docs or in the readme, it's on my to do list, to properly document the build steps